src/sentry/api/base.py

```diff
@@ -156,14 +162,14 @@ def build_cursor_link(self, request: Request, name, cursor):
             mutable_query_dict.pop("cursor")
             querystring = mutable_query_dict.urlencode()
 
-        base_url = absolute_uri(urlquote(request.path))
+        base_url = request.build_absolute_uri()
```

Comment:
we have no tests for this case, so lets risk breakage here to uncover if theres a genuine problem. its possible our absolute_uri function does not handle the same conditions that build_absolute_uri does, but I cannot recall/come up with a scenario where we need to quote the path here.

the examples where this _might_ need to happen woudl be like if we had `/foo/bar/cool:story/`, and need to re-encode that to pass it as a new url

Reply:
retaining behavior as its cetainly expected to quote. added a test